pkg/api/pullrequestpayloadtest/v1/ci.openshift.io_pullrequestpayloadtests.yaml

```diff
@@ -0,0 +1,80 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
+  creationTimestamp: null
+  name: pullrequestpayloadtests.ci.openshift.io
```

Comment:
Should this live in the release controller?

Reply:
This resource looks more directly related to CI at the moment.